### DISCUSSION TITLE
Meshing An Exodus File Using MOOSE

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#25723] [Q&A Meshing] [answered]
Can an unmeshed exodus file be meshed using online generation in MOOSE?

**Comment by lindsayad** [comment]
What do you mean by unmeshed? Exodus stores data on nodes or elements so there is already mesh construct in it. You can use a FileMeshGenerator to load an exodus mesh

**Reply by TLWise** [reply]
Ah, I see what your question implies. Cubit does not allow me to save the geometry without meshing it as a .e file. Can I mesh the geometry with a minimal amount of meshing in Cubit, import it into MOOSE, delete the previous mesh and then use online generation meshing in MOOSE to define a new mesh?

**Reply by GiudGiud** [reply]
Hello

which dimension is your geometry?
is this a simple geometry?

Guillaume

**Reply by TLWise** [reply]
![Screenshot from 2023-10-11 14-06-06](https://github.com/idaholab/moose/assets/39968866/5bb161cd-02b1-4746-866b-a0bbec7e752b)

**Reply by GiudGiud** [reply, marked as answer]
I would say just create it directly from scratch in MOOSE if you are trying to use MOOSE mesh generators

the reactor module is meant to model these lattice systems

**Reply by TLWise** [reply]
@GiudGiud , I applied the method suggested above. Thank you for your guidance. The numerical results that I am getting by using the basic method, appears to imply that the boundary condition that I am applying to the overall outer boundary is being applied equally to each individual location outer boundary. I'm applying equal power density to each circular region, this an expectation for the highest temperature at the center of the array. It appears a symmetry condition is being applied to each section equally. I have included a pic of the result. Is there a way to prevent the application of this symmetry boundary condition?
![Screenshot from 2023-10-12 14-06-58](https://github.com/idaholab/moose/assets/39968866/846d4a26-4d02-4806-a5d3-e3ea7c6f9ebc)

**Reply by GiudGiud** [reply]
Looks like you ll need to rename the outer boundary to apply your condition only there
You use other mesh generators (sidesetfromnormal, sidesetaround etc) to do that

You can plot the sidesets in paraview to see where they are defined.
